template/index.html

```diff
@@ -4,6 +4,24 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- Primary Meta Tags -->
+<title>Portfolio Template - Open source</title>
+<meta name="title" content="Portfolio Template - Open source">
+<meta name="description" content="Build or customize your portfolio website using this super easy template. Free to use if you are a member of the community or a contributor to this project">
+
+<!-- Open Graph / Facebook -->
+<meta property="og:type" content="website">
+<meta property="og:url" content="https://cpro-portfolio-html.netlify.app/">
+<meta property="og:title" content="Portfolio Template - Open source">
+<meta property="og:description" content="Build or customize your portfolio website using this super easy template. Free to use if you are a member of the community or a contributor to this project">
+<meta property="og:image" content="https://user-images.githubusercontent.com/62628408/150613011-b78a7f5a-0af1-4312-aab0-0022e7258693.png">
+
+<!-- Twitter -->
+<meta property="twitter:card" content="summary_large_image">
+<meta property="twitter:url" content="https://cpro-portfolio-html.netlify.app/">
```

Comment:
Here too. So when someone clicks the link, it will take them to the github page and not the template itself. 

https://github.com/CommunityPro/portfolio-html